locales-cs-CZ.xml

```diff
@@ -251,6 +251,7 @@
     </term>
 
     <!-- VERB ROLE FORMS -->
+    <term name="container-author" form="verb"/>
```

Comment:
I guess that must be what we do then. To the question, I have seen
differential treatment of singletons in HTML parsers (some will drop the
node), but XML should (should) be consistent. I'll check what citeproc-js
does with it later, fwiw.
On Oct 16, 2015 05:02, "Rintze M. Zelle" <notifications@github.com> wrote:

> In locales-cs-CZ.xml
> <https://github.com/citation-style-language/locales/pull/122#discussion_r42172764>
> :
>
> > @@ -251,6 +251,7 @@
> >      </term>
> >
> >      <!-- VERB ROLE FORMS -->
> > +    <term name="container-author" form="verb"/>
>
> http://docs.citationstyles.org/en/stable/specification.html#locale-fallback
> says
>
> Fallback stops once a localizable unit has been found. For terms, this
> even is the case when they are defined as empty strings (e.g. <term
> name="and"/> or <term name="and"></term>).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/citation-style-language/locales/pull/122/files#r42172764>
> .
>